src/main/kotlin/prison/model.kt

```diff
@@ -15,8 +15,15 @@ fun prisonModel(model: Model) {
   model.setImpliedRelationshipsStrategy(
     CreateImpliedRelationshipsUnlessAnyRelationshipExistsStrategy())
 
+  val spo = model.addPerson("Senior Prison Offender Manager", "manages service users and offender managers")
+  val pom = model.addPerson("Prison Offender Manager", "responsible for the service users in their prison")
+  pom.interactsWith(spo, "managed by")
+
   HmmpsAuth(model)
   Delius(model)
   NDH(model).system
   PATHFINDER(model)
+
+  spo.uses(OffenderManagementInCustody.allocationManager, "look at unallocated service users coming from court in")
+  pom.uses(OffenderManagementInCustody.allocationManager, "look at service users who need handing over to community in")
```

Comment:
@spikeheap I wanted to use these within the `defineRelationships`, but that needed
- referencing the people by name, which is not available there (needs a `model` which I didn't want to pass through)
- creating a new holder object for 'internal teams' (which I didn't want to include here as it sounds like a significant thing)

For the time being, I opted to define these person-container relationships here to tackle it later.